goon_test.go

```diff
@@ -892,92 +948,142 @@ func TestMigration(t *testing.T) {
 	}
 
 	// Run migration tests with both IgnoreFieldMismatch on & off
-	for i := 0; i < 2; i++ {
-		IgnoreFieldMismatch = (i == 0)
-
-		// Clear all the caches
-		g.FlushLocalCache()
-		memcache.Flush(c)
-
-		// Get it back, so it's in the cache
-		migA = &MigrationA{Parent: parentKey, Id: 1}
-		if err := g.Get(migA); err != nil {
-			t.Errorf("Unexpected error on Get: %v", err)
+	for _, IgnoreFieldMismatch := range []bool{true, false} {
```

Comment:
This loop declares a local scope variable called `IgnoreFieldMismatch` which hides the global one. I think the easiest fix is to change `:=` to `=`.

Reply:
Well, that's a silly one...
Let me fix that quickly.

Reply:
Fixed by 7bb8fe0